addon/package.json

```diff
@@ -149,7 +149,7 @@
   "typesVersions": {
     "*": {
       "*": [
-        "public-types/@ember/test-helpers/*"
+        "./public-types/*"
```

Comment:
@gitKrystan @chriskrycho there was a tiny variation between both of your suggestion, should it be `./public-types/*` or `public-types/*`?

Reply:
Either should work.